openrtx/include/core/cps.h

```diff
@@ -45,27 +44,23 @@
  * present in the codeplug binary data.
  */
 #define MAX_TONE_INDEX 50
-static const uint16_t ctcss_tone[MAX_TONE_INDEX] =
-{
-    670, 693, 719, 744, 770, 797, 825, 854, 885, 915, 948, 974, 1000, 1034,
-    1072, 1109, 1148, 1188, 1230, 1273, 1318, 1365, 1413, 1462, 1514, 1567,
-    1598, 1622, 1655, 1679, 1713, 1738, 1773, 1799, 1835, 1862, 1899, 1928,
-    1966, 1995, 2035, 2065, 2107, 2181, 2257, 2291, 2336, 2418, 2503, 2541
-};
+static const uint16_t ctcss_tone[MAX_TONE_INDEX] = {
+    670,  693,  719,  744,  770,  797,  825,  854,  885,  915,
+    948,  974,  1000, 1034, 1072, 1109, 1148, 1188, 1230, 1273,
+    1318, 1365, 1413, 1462, 1514, 1567, 1598, 1622, 1655, 1679,
+    1713, 1738, 1773, 1799, 1835, 1862, 1899, 1928, 1966, 1995,
+    2035, 2065, 2107, 2181, 2257, 2291, 2336, 2418, 2503, 2541};
 
 /**
  * Data structure defining an analog-specific channel information such as tones.
  */
 typedef struct
 {
-    uint8_t rxToneEn : 1,   //< RX CTC/DCS tone enable
-            rxTone   : 7;   //< RX CTC/DCS tone index
-    uint8_t txToneEn : 1,   //< TX CTC/DCS tone enable
-            txTone   : 7;   //< TX CTC/DCS tone index
-}
-__attribute__((packed)) fmInfo_t; // 2B
-
-
+    uint8_t rxToneEn : 1,           //< RX CTC/DCS tone enable
+        rxTone       : 7;           //< RX CTC/DCS tone index
```

Comment:
All the fields should stay aligned, like it was before. This makes them more readable

Reply:
Yes!

Reply:
Haven't cracked how to do this yet.